modules/onetagBidAdapter.js

```diff
@@ -87,7 +88,8 @@ function buildRequests(validBidRequests, bidderRequest) {
   const connection = navigator.connection || navigator.webkitConnection;
   payload.networkConnectionType = (connection && connection.type) ? connection.type : null;
   payload.networkEffectiveConnectionType = (connection && connection.effectiveType) ? connection.effectiveType : null;
-  payload.fledgeEnabled = Boolean(bidderRequest && bidderRequest.fledgeEnabled)
+  payload.fledgeEnabled = Boolean(bidderRequest && bidderRequest.fledgeEnabled);
+  payload.coppa = config.getConfig('coppa') === true;
```

Comment:
please read this from the request object, we're hoping to get rid of bidders calling getConfig in the future.